vendor/bat-native-ads/src/bat/ads/internal/ml/transformation/lowercase.cc

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <algorithm>
+#include <memory>
+#include <string>
+
+#include "base/strings/string_util.h"
+#include "bat/ads/internal/ml/data/text_data.h"
+#include "bat/ads/internal/ml/transformation/lowercase.h"
```

Comment:
nit: `lowercase_transformation` for the filename

Reply:
You mean only changing file name? Class name should remain the same?

Reply:
Should I change `normalization` and `hashed_ngrams` in the same way?

Reply:
yes please, thanks

Reply:
Done